src/transformers/pipelines.py

```diff
@@ -1195,7 +1196,8 @@ def __init__(
         framework: Optional[str] = None,
         args_parser: ArgumentHandler = None,
         device: int = -1,
-        topk=5,
+        top_k=5,
+        topk: Optional[int] = None,
```

Comment:
Usually we put the deprecated arguments in the kwargs so they are not visible anymore.

Reply:
Ok will do